<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1634</width>
    <height>943</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(226, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>900</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-620</y>
         <width>1593</width>
         <height>1518</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1500</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPlainTextEdit" name="name">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>90</y>
             <width>1211</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(153, 229, 229);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>800</y>
             <width>141</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Category:</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>800</y>
             <width>631</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;Elephant&quot;;</string>
           </property>
           <item>
            <property name="text">
             <string>Travel</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Food</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sports</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fashion</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Personal</string>
            </property>
           </item>
          </widget>
          <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
            <rect>
             <x>1230</x>
             <y>90</y>
             <width>131</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 28pt &quot;&quot;;
text-decoration: underline;
font: 20pt &quot;Eras Bold ITC&quot;;</string>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>830</x>
             <y>800</y>
             <width>131</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 28pt &quot;&quot;;
text-decoration: underline;
font: 20pt &quot;Eras Bold ITC&quot;;</string>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
          <widget class="QTableWidget" name="tableWidget">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>890</y>
             <width>1391</width>
             <height>461</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <column>
            <property name="text">
             <string>Author</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Title</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Content</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Like</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Dislike</string>
            </property>
           </column>
          </widget>
          <widget class="QPushButton" name="pushButton_4">
           <property name="geometry">
            <rect>
             <x>1320</x>
             <y>1360</y>
             <width>111</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 28pt &quot;&quot;;
text-decoration: underline;
font: 15pt &quot;Eras Bold ITC&quot;;</string>
           </property>
           <property name="text">
            <string>View</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>1490</x>
             <y>10</y>
             <width>81</width>
             <height>33</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>200</y>
             <width>1591</width>
             <height>531</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/91810/Pictures/Screenshots/sawantttt.png</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>451</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 28pt &quot;&quot;;
text-decoration: underline;
font: 30pt &quot;Eras Bold ITC&quot;;</string>
           </property>
           <property name="text">
            <string>EXPLORE BLOGS</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>740</y>
             <width>741</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 28pt &quot;&quot;;
text-decoration: underline;
font: 20pt &quot;Eras Bold ITC&quot;;</string>
           </property>
           <property name="text">
            <string>Search Through Your Favourite Categories:</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
